src/constructs/ec2/security-groups/wazuh.ts

```diff
@@ -10,7 +10,6 @@ export class GuWazuhAccess extends GuBaseSecurityGroup {
     super(scope, "WazuhSecurityGroup", {
       vpc,
       description: "Wazuh agent registration and event logging",
-      overrideId: true,
```

Comment:
This seems like an odd default, given that the logical id was hardcoded... However, I guess anyone who had CloudFormed a resource called `WazuhSecurityGroup` would have inherited it by default before. Given how fiddly it is to replace security groups, do we need to provide a way to retain your existing Wazuh security group via the constructor?

Reply:
Maybe the Wazuh construct should hard-code the logicalId? It's a singleton, so it shouldn't matter too much.

Reply:
I guess hardcoding will work fine as long as everyone who CloudForms this SG copy+pastes the [example logical id](https://github.com/guardian/security-hq/blob/main/hq/markdown/wazuh.md#outbound-traffic-on-ports-1514-and-1515) (seems reasonably likely!).

Reply:
The plot thickens!

An update to a security group's description is [destructive](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group.html#cfn-ec2-securitygroup-groupdescription).

> Update requires: Replacement

The description we're using in this library differs from SecurityHQ's guide ("Wazuh agent registration and event logging" vs "Allow outbound traffic from wazuh agent to manager") 😱 .

I think it'll be easiest to change this library as adoption is still relatively low therefore we can more easily provide support to teams currently using it.

Reply:
https://github.com/guardian/cdk/pull/402